### ISSUE TITLE
[1.12.1] Shift-clicking unfilled Summoning Pendulum Crash

### ISSUE THREAD

**Issue by mewzurui** [lumien231/Random-Things#243]
Hi,

A crash is produced when shift clicking a semi-filled summoning pendulum into an inventory. The filled ones have no crash. I have attached the crash log.

[crash-2017-09-24_10.31.13-client.txt](https://github.com/lumien231/Random-Things/files/1327387/crash-2017-09-24_10.31.13-client.txt)

**Comment by lumien231** [comment]
Can you reproduce this with only RT installed?

**Comment by mewzurui** [comment]
No, it does not crash alone with forge or optifine, im guessing it may be another mod, the entities inside were from animania. Let me know if it is another mod or if it can be fixed on this end please.

**Comment by ProsperCraft** [comment]
Same crash - we suspect it is due to it containing an entity that we removed from the game.

This is on a multiplayer server, the user just opened it to see what was in it.
https://paste.dimdev.org/yunikipome.mccrash
reproduced here -
https://paste.dimdev.org/jicenoteza.mccrash

**Comment by Dooglesmile** [comment]
I'm getting the same problem and crash to desktop when holding shift on the summoning pendulum to list what creatures are contained inside it when I have four captured Twilight Forest creatures in it. 

Holding shift to view what creatures are contained in one with five twilight forest creatures inside shows the list properly with no issues.

[Crash report](https://paste.dimdev.org/kujeriweta.pl)

Oh, it does let me release the creatures if I don't try to view the list of what are captured in the pendulum.

Edit:
After further testing it seems that, in my case at least, when I was trying to grab the creatures, if I missed on the first try it was possibly getting a blank entry into the summoning pendulum, as when I filled the one I had that was causing a crash, it still caused a crash until I fully emptied it, which took six clicks to do rather than just five to put all five creatures into the world.
Then when I collected the creatures back again (even just three of them), holding shift over the pendulum did not cause a crash.
